.github/workflows/test.yml

```diff
@@ -29,7 +29,7 @@ jobs:
       - name: Run ansible-lint
         uses: ansible/ansible-lint@main
         with:
-          requirements_file: ../requirements.yml
+          requirements_file: requirements.yml
           working_directory: src
```

Comment:
unrelated, but this means we never run lint on development playbooks.

while certainly less important, I'd still prefer if we have them linted.